core/interfaces/src/main/kotlin/app/aaps/core/interfaces/aps/OapsProfile.kt

```diff
@@ -47,5 +47,29 @@ data class OapsProfile(
     //DynISF only
     var variable_sens: Double,
     var insulinDivisor: Int,
-    var TDD: Double
+    var TDD: Double,
```

Comment:
Create `data class OapsProfileAutoIsf : OapsProfile`
this makes the profile value confusing for other algs

Reply:
I tried, but difficulty was within APSResult interface and DetermineBasalResult class that also include OapsProfile for input information (so your request was not limited to AutoIsf plugin and associated unit tests, it also has side impacts on database with probably an additional field for profileJsonAutoIsf and OpenAPSFragment...)
=> I finally prefered to keep OapsProfile class for that reason (looks to me globally much more simple to keep OapsProfile class with default values for dedicated AutoISF values...)

![image](https://github.com/nightscout/AndroidAPS/assets/52934600/74aea42a-de9a-4c8e-9d6a-7cfe8d20cb5a)
![image](https://github.com/nightscout/AndroidAPS/assets/52934600/fd0e346e-28b2-4540-a6cc-8d4a40298461)

Do you confirm this request ?

Reply:
I added the dedicated OapsProfileAutoIsf file (without modifying database).
😉